public/components/event_analytics/explorer/log_explorer.tsx

```diff
@@ -110,7 +116,7 @@ export const LogExplorer = ({
 
   const addNewTab = async (where: string) => {
     // get a new tabId
-    const tabId = uniqueId(TAB_ID_TXT_PFX);
+    const tabId = htmlIdGenerator(TAB_ID_TXT_PFX)();
```

Comment:
what was the problem with uniqueId?

Reply:
Since we are now presisting data between refreshes, the previous tab status remains, but refreshing causes uniqueId to reset, therefore it generates same 'query-panel-1', 'query-panel-2', '...' ids again but such tabs have already existed which causes bug. htmlIdGenerator generates universally unique id.

Reply:
Also, let's stick to EUI standards. This will help us to shift to OUI smoothly later.